dts/bindings/mipi-dbi/mipi-dbi-controller.yaml

```diff
@@ -18,3 +18,13 @@ properties:
   "#size-cells":
     required: true
     const: 0
+
+  data-bus-width:
```

Comment:
I think rather than adding the data bus property to the display, we should define multiple MIPI modes for each data bus width. Something like the following:

```
#define MIPI_DBI_MODE_8080_BUS_16_BIT 0x4
#define MIPI_DBI_MODE_8080_BUS_9_BIT 0x5
#define MIPI_DBI_MODE_8080_BUS_8_BIT 0x6
```

I see two benefits to this:
- avoid duplication of bus width encoding in each display driver (I suspect this problem won't be unique to the ST7796S)
- allow MIPI DBI drivers to reconfigure their bus width at runtime (I'm aware that due to the SDK driver structure this won't work for FlexIO, but I suspect most hardware will support runtime reconfiguration like this)

Reply:
@danieldegrasse, I have made this change.